src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0; net471</TargetFrameworks>

```

Comment:
This gives us [full support for .NET Standard 2.0](https://devblogs.microsoft.com/dotnet/announcing-the-net-framework-4-7-1/) correct? And the Azure Functions host supports this framework version?

Reply:
The Functions host does support this framework. Not sure about the "full support" or not.